packages/bsky/src/hydration/graph.ts

```diff
@@ -195,4 +195,24 @@ export class GraphHydrator {
     })
     return { followers: res.followers, cursor: res.cursor }
   }
+
+  async getKnownFollowers(input: {
```

Comment:
This is a bit confusing because there's a method with the same name on the actor hydrator & it's not really clear the difference

Since this is only used for building the skeleton for the `getKnownFollowers` route & not actually for hydration, I might move this logic over to the route handler itself